src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,13 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public abstract class Command {
+
+    public abstract String getResponse(TaskList tasks, Ui ui, Storage storage) throws DukeException;
+
+    public abstract boolean isExitCommand();
```

Comment:
Though you probably already satisfy the "Javadoc more than 50% of public classes and methods" requirements, a useful tip is that by adding Javadocs to a class with methods to be overriden such as the abstract Command class, these Javadocs will appear in Quick Tip windows when users however over overriden implementations.

E.g. Writing a javadoc for getResponse in the Command class will show users the Javadoc when they activate Quick Tip (Ctrl + Q in IntelliJ) for an overriden method in the DeleteTaskCommand class even though there isnt a Javadoc written in the DeleteTaskCommand class.

Reply:
Thanks for the tip!!